sonar-python-plugin/src/main/java/org/sonar/plugins/python/ExternalIssuesSensor.java

```diff
@@ -62,7 +78,40 @@ private void logUnresolvedInputFiles(Set<String> unresolvedInputFiles) {
     logger().warn("Failed to resolve {} file path(s) in " + linterName() + " report. No issues imported related to file(s): {}", unresolvedInputFiles.size(), fileList);
   }
 
-  protected abstract void importReport(File reportPath, SensorContext context, Set<String> unresolvedInputFiles);
+  private void logFileCantBeRead(Exception e, File reportPath) {
+    logger().error("No issues information will be saved as the report file '{}' can't be read. " +
+      e.getClass().getSimpleName() + ": " + e.getMessage(), reportPath, e);
```

Comment:
It's a little weird to me that we are mixing string formatting and string concatenation here.